Vergen failing to compile as dependency when updating to 8.2.3 from 8.2.1

Recently dependabot tried to update vergen in a project from 8.2.1 to 8.2.3. It is failing at compile stage with error 
```sh
   Compiling vergen v8.2.3
error[E0599]: no method named `checked_to_offset` found for struct `OffsetDateTime` in the current scope
   --> .../.cargo/registry/src/index.crates.io-6f17d22bba15001f/vergen-8.2.3/src/feature/git/cmd.rs:651:54
    |
651 |                         let local_offset = no_offset.checked_to_offset(local).unwrap_or(no_offset);
    |                                                      ^^^^^^^^^^^^^^^^^ help: there is a method with a similar name: `to_offset`

```

The vergen dependency in cargo.tom is `vergen = {version ="8.2.1", features =["git","gitcl"]}`.

The dependabot PR on which this is failing is https://github.com/containers/youki/pull/2111 actual CI https://github.com/containers/youki/actions/runs/5451984710/jobs/9918894204?pr=2111

I also tried building locally and it fails with the same error.
Is there some config issue such as feature selection issue from my side, or anything else?

Thanks!

--- a/vergen/Cargo.toml
+++ b/vergen/Cargo.toml
@@ -44,7 +44,7 @@ git2-rs = { version = "0.17.1", package = "git2", optional = true, default-featu
 gix = { version = "0.48.0", optional = true, default-features = false }
 rustc_version = { version = "0.4.0", optional = true }
 sysinfo = { version = "0.29.0", optional = true, default-features = false }
-time = { version = "0.3.20", features = [
+time = { version = "0.3.22", features = [
     "formatting",
     "local-offset",
     "parsing",

